### ISSUE TITLE
Disable operator pending mappings

### ISSUE THREAD

**Issue by sassanh** [SmiteshP/nvim-navbuddy#41]
Thanks for this handy plugin!

Currently when I press `c` to comment a piece of code, it happens after some delay, because Neovim waits to see if I hit any other key after `c` as it goes into operator pending mode (`c` is the operator).

I have seen some plugins like Mason, Lazy, Packer, etc somehow disable operator pending keymaps or somehow put more priority for their own plugin (I think Lazy directly listens to key events).

It would be nice if we could have something similar for navbuddy.

**Comment by SmiteshP** [comment]
Why does it do into operator pending mode when navbuddy is already remapping the key 🤔 Will have to look into this.

**Comment by rcsalomao** [comment]
First I would like to thank you for this plugin! It's really good.

I believe what's happening is that even if you are mapping the key (In this case locally), it might conflict with any other global keybinding.
I too am experiencing the same behaviour as @sassanh with the comment keybind.
I make some tests and the keybinding moment is in display.lua file line 238 correct? I included on the :map(...) function the option { nowait=true } on it and it seems to have fixed and produce the desired behaviour.
I just made a PR (#54 ) with the fix.
